sources/packages/web/src/services/http/dto/EducationProgram.dto.ts

```diff
@@ -143,6 +144,8 @@ export class EducationProgramAPIInDTO {
   programDeclaration: boolean;
   @Expose()
   fieldOfStudyCode: number;
+  isBCPrivate: boolean;
```

Comment:
Are these required in the Web side or they were added just for the sake of keeping the DTO the same from the API?

Reply:
They are not required in the web side, and they were added just for keeping the DTO the same from the API.